lib/ios-deploy.js

```diff
@@ -126,8 +126,8 @@ class IOSDeploy {
     }
   }
 
-  async installApp (app, timeout) {
-    await this.install(app, timeout);
+  async installApp(...args) {
```

Comment:
Please keep the space before parentheses. The linter verifies that

Reply:
tnx, done